_releaser/cloudfront-lambda-redirects.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+exports.handler = (event, context, callback) => {
+    //console.log("event", JSON.stringify(event));
+    const request = event.Records[0].cf.request;
```

Comment:
Just a question; does JavaScript now allow some "doc" or "type hint" to make my IDE understand `event`? I noticed my IDE didn't understand the type, so added a "wiggly-line" below `Records`;

<img width="616" alt="Screenshot 2022-08-17 at 12 29 00" src="https://user-images.githubusercontent.com/1804568/185097738-c41ee1ca-6846-4c33-8aa1-58065a853f24.png">


So was wondering if there's some metadata we can add to make it understand the type (definitely not urgent, just wondering)

Reply:
I was also looking at this too and without a proper DSL we can't unfortunately and I don't think AWS provides one.

Reply:
Gotcha; no problem (just curious 😅)